Backported the Reconstruct world space positions example (PR #2849). Plus other changes: upgrade instructions, requirements, what's new.

oleks-k opened the pull request:
* Backported the Reconstruct world space positions example.
  Original PR: https://github.com/Unity-Technologies/Graphics/pull/2849
* Updated the upgrade instructions (minor update, no changes required).
* Updated the Requirements page.
* Updated the What's new page (just a maintenance release).

phi-lira reviewed (changes requested):
`com.unity.render-pipelines.universal/Documentation~/writing-custom-shaders-urp.md.bak` file seems a mistake. 

I didnt' review content of PR as this is a backport, just overall review of files.

oleks-k commented:
> `com.unity.render-pipelines.universal/Documentation~/writing-custom-shaders-urp.md.bak` file seems a mistake.
> 
> I didnt' review content of PR as this is a backport, just overall review of files.

Thanks! I've fixed that (also fixed the settings in my merge program).
